apps/admin/api-routes/base-url.ts

```diff
@@ -0,0 +1 @@
+export const BASE_URL = process.env.BASE_URL || 'http://127.0.0.1:8000/api/';
```

Comment:
```suggestion
export const BASE_URL = process.env.BASE_URL || 'http://127.0.0.1:8000/api/';
```
will it be better to have diff base url's for diff env's ?

BASE_URL_LOCAL
BASE_URL_STAGING
BASE_URL_PROD

Reply:
@yaju09 we don't have any plan for infrastructure. So let's not plan for environment related works.

Reply:
@yogai53 Then should I hard code the url for now ?

Reply:
@yogai53 got it Yoga

Reply:
@sidarth-23 the above code is fine in that case.